docs/release-notes/eclair-vnext.md

```diff
@@ -0,0 +1,171 @@
+# Eclair vnext
+
+<insert here a high-level description of the release>
+
+## Major changes
+
+### ZMQ changes
+
+Eclair previously used ZMQ to receive full blocks from Bitcoin Core.
+In this release, we instead switch to receive only block hashes over ZMQ.
+This will save bandwidth and improve support for deployments with a remote Bitcoin Core node.
+
+:warning: When updating eclair, you need to update your `bitcoin.conf` to have your Bitcoin Core node send block hashes via ZMQ.
+
+The previous configuration was:
+
+```conf
+zmqpubrawblock=tcp://127.0.0.1:29000
+```
+
+You must remove that line from your `bitcoin.conf` and replace it with:
+
+```conf
+zmqpubhashblock=tcp://127.0.0.1:29000
+```
+
+### Beta support for anchor outputs
+
+Anchor outputs is still disabled by default, but users willing to try it can activate it by adding the following line to `eclair.conf`:
+
+```conf
+eclair.features.option_anchors_zero_fee_htlc_tx = optional
+```
+
+Once activated, eclair will keep the commitment feerate below 10 sat/byte regardless of the current on-chain feerate and will not close channels when there is a feerate mismatch between you and your peer.
+
+You can modify that threshold by setting `eclair.on-chain-fees.feerate-tolerance.anchor-output-max-commit-feerate` in your `eclair.conf`.
+Head over to [reference.conf](https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/resources/reference.conf) for more details.
+
+In case the channel is unilaterally closed, eclair will use CPFP and RBF to ensure that transactions confirm in a timely manner.
+You **MUST** ensure you have some utxos available in your Bitcoin Core wallet for fee bumping, otherwise there is a risk that an attacker steals some of your funds.
+
+Do note that anchor outputs may still be unsafe in high-fee environments until the Bitcoin network provides support for [package relay](https://bitcoinops.org/en/topics/package-relay/).
+
+### Path-finding improvements
+
+This release contains many improvements to path-finding and paves the way for future experimentation.
+It adds support for A/B testing and various refinements of the previous heuristics.
+
```

Comment:
```suggestion
This release adds a new path-finding setting: a virtual cost per hop that can be used to penalize long paths.

It also adds support for A/B testing to find out which values work best.
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1951/commits/26fe219e208a98326dfd260d58e24b70d1e5358c